evalai/utils/challenges.py

```diff
@@ -87,7 +87,35 @@ def display_ongoing_challenge_list():
     Displays the list of ongoing challenges from the backend
     """
     url = "{}{}".format(API_HOST_URL, URLS.challenge_list.value)
-    display_challenges(url)
+
+    header = get_request_header()
```

Comment:
This http request pattern seems to be quite common. @isht3 can you please open an issue to abstract this out and mark it as `low-priority`? This could be something you could work on after GSOC or for someone else to work out.

Reply:
Yes, agreed. Except for a few functions and the error responses, the structure is basically the same. #81